packages/core/content-releases/server/src/utils/index.ts

```diff
@@ -1,6 +1,19 @@
+import type { LoadedStrapi, UID } from '@strapi/types';
+
 export const getService = (
   name: 'release' | 'release-validation',
   { strapi } = { strapi: global.strapi }
 ) => {
   return strapi.plugin('content-releases').service(name);
 };
+
+export const getAllowedContentTypes = ({ strapi, userAbility }: { strapi: LoadedStrapi, userAbility: any }) => {
+  // We get the content types that have draft and publish enabled and that the user can read
```

Comment:
I think it could be valuable to place the function description like this:
```suggestion
/**
 * Gets the content types that have draft and publish enabled and that the user can read
 */
export const getAllowedContentTypes = ({ strapi, userAbility }: { strapi: LoadedStrapi, userAbility: any }) => {
```

Then when you hover over it anywhere you see it ✨ :

![Screenshot 2023-11-28 at 11 41 27](https://github.com/strapi/strapi/assets/26598053/722c2bac-cd83-4db6-8692-00144e3c8000)